src/translations/config.yaml

```diff
@@ -0,0 +1,3 @@
+cultural_date_format: "%d/%m/%Y"
+cultural_bignumer_mark: " "
+cultural_punctuation_mark: ","
```

Comment:
is this format correct? seems that this is the Polish config

Reply:
This file isn't used by translations, it's a default from the package to avoid the warning.

Reply:
hm, weird; maybe we should update the package so that it does not show warnings? ok for now